### PR TITLE
Consolidated <bindings> element documentation

### LINKED ISSUE #9289: Consolidate WCF binding topic
## Consolidate WCF binding topics

The [\<binding> article](https://docs.microsoft.com/en-us/dotnet/framework/misc/binding) does not sync with the TOC and contains information more appropriate to the configuration file reference. It should be merged into the [\<bindings> article in the Configuration File Schema reference](https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/wcf/bindings), and requests for the article in the **misc** directory should redirect to the configuration file schema article. 

For the most part, the content of these two articles is identical. Any unique content in the article in the **misc** directory should be merged into the single article.

### PR DESCRIPTION
**Pull request by rpetrusha**
## Consolidated \<bindings> element documentation

Fixes #9289

### CODE REVIEW

File: docs/framework/configure-apps/file-schema/wcf/bindings.md [hunk 1]
```diff
@@ -1,28 +1,38 @@
 ---
 title: "&lt;bindings&gt;"
-ms.date: "03/30/2017"
+ms.date: "01/22/2018"
 ms.assetid: b62cd369-5409-4030-8490-9759a462dd3a
 ---
 # &lt;bindings&gt;
-This section holds a collection of standard and custom bindings. Each entry is a `binding` element that can be identified by its unique `name`. Services use bindings by linking them using the `name`. Starting with [!INCLUDE[netfx40_short](../../../../../includes/netfx40-short-md.md)], bindings and behaviors are not required to have a name. For more information about default configuration and nameless bindings and behaviors, see [Simplified Configuration](../../../../../docs/framework/wcf/simplified-configuration.md) and [Simplified Configuration for WCF Services](../../../../../docs/framework/wcf/samples/simplified-configuration-for-wcf-services.md).  
+
+---
```
Comment (mairaw) [inline docs/framework/configure-apps/file-schema/wcf/bindings.md]: you have an extra yml section here

File: docs/framework/configure-apps/file-schema/wcf/bindings.md [hunk 2]
```diff
@@ -33,10 +43,11 @@ This section holds a collection of standard and custom bindings. Each entry is a
...
-- [Custom Bindings](../../../../../docs/framework/wcf/extending/custom-bindings.md)
-- [\<customBinding>](../../../../../docs/framework/configure-apps/file-schema/wcf/custombinding.md)
+
+- <xref:System.ServiceModel.Configuration.BindingsSection?displayProperty=nameWithType>  
+- <xref:System.ServiceModel.Channels.Binding?displayProperty=nameWithType>  
+- <xref:System.ServiceModel.Channels.BindingElement?displayProperty=nameWithType>  
+- [Bindings](../../../../../docs/framework/wcf/bindings.md)  
+- [Custom Bindings](../../../../../docs/framework/wcf/extending/custom-bindings.md)  
+- [\<customBinding>](../../../../../docs/framework/configure-apps/file-schema/wcf/custombinding.md)  
 - [\<binding>](../../../../../docs/framework/misc/binding.md)
```
Comment (mairaw) [inline docs/framework/configure-apps/file-schema/wcf/bindings.md]: you should see if you have more incoming links like this to the old topic and update them to point to this new one. This one needs to be removed though.

File: docs/framework/configure-apps/file-schema/wcf/bindings.md [hunk 3]
```diff
@@ -1,28 +1,38 @@
...
+ 
  System-provided bindings hide the complexity of the WCF messaging stack. Applications using system-provided bindings do not require full control over the stack. The attributes exposed on each system-provided binding are the ones most appropriate for the usage scenario the binding addresses.  
   
  The configuration section for each system-provided binding can define several configurations used to configure the binding. Each configuration is identified by a unique name.  
   
- It is not possible to add elements or attributes to a system-provided binding. To do so, you should implement a custom binding as described in the "Custom Binding" section of this topic. It is possible to define a custom binding that mimics a system-provided binding perfectly and adds a few settings the user application wants to have control over.  
+ It is not possible to add elements or attributes to a system-provided binding. To do so, you should implement a custom binding as described in the [Custom Bindings](#custom-bindings)section. It is possible to define a custom binding that mimics a system-provided binding perfectly and adds a few settings the user application wants to have control over.
```
Comment (mairaw) [inline docs/framework/configure-apps/file-schema/wcf/bindings.md]:
```suggestion
 It isn't possible to add elements or attributes to a system-provided binding. To do so, you should implement a custom binding as described in the [Custom bindings](#custom-bindings) section. It's possible to define a custom binding that
  mimics a system-provided binding perfectly and adds a few settings the user application wants to have control over.  
```

**Review by mairaw — APPROVED** [review]
LGTM but there a few things to fix before merging

**Comment by mairaw** [comment]
There are more conflicts here @rpetrusha

**Comment by BillWagner** [comment]
Closing. These commits were incorporated into #10694 to resolve the merge conflicts.

/cc @rpetrusha

**Pull request closed without merging**